Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp

```diff
@@ -20,8 +22,32 @@ DivBFunctor::operator()(amrex::MultiFab& mf_dst, int dcomp, const int /*i_buffer
     constexpr int ng = 1;
     // A cell-centered divB multifab spanning the entire domain is generated
     // and divB is computed on the cell-center, with ng=1.
-    amrex::MultiFab divB( warpx.boxArray(m_lev), warpx.DistributionMap(m_lev), 1, ng );
+    amrex::MultiFab divB( warpx.boxArray(m_lev), warpx.DistributionMap(m_lev), 2*warpx.n_rz_azimuthal_modes-1, ng );
```

Comment:
More readable would be:
```suggestion
    amrex::MultiFab divB( warpx.boxArray(m_lev), warpx.DistributionMap(m_lev), n_rz_mode_inds, ng );
```

and I think we need to guard this for RZ only (otherwise one component) :)